.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: daily
```

Comment:
Hm, perhaps `weekly` (or `monthly`) would do?

Also, beware of https://github.com/ocaml/ocaml/blob/12e34c5d1df0fd1f0321dcb6cedbda2d9cbf1cd7/.github/workflows/build.yml#L275-L276
which is using `checkout@v1` on purpose as the i386 workflows don't work on newer versions.